headers/csps/chains/arbitrum.ts

```diff
@@ -0,0 +1,14 @@
+import { loadEnv } from 'vite'
+
+import type { Csp } from '../../types'
+
+const env = loadEnv(process.env.NODE_ENV || 'development', process.cwd(), '')
+
+export const csp: Csp = {
+  'connect-src': [
+    env.VITE_ARBITRUM_NODE_URL!,
```

Comment:
@NeOMakinG `loadEnv()` returns a `Record<string, string>` so we should remove the non-null assert from all CSPs 
<img width="743" alt="image" src="https://github.com/user-attachments/assets/5dc7c1fa-f807-40a7-af9d-ab04142e0fb5" />

see https://github.com/shapeshift/web/pull/8918, not a blocker if you don't want to do it as part of this PR, I'll tackle it in 8918 !

Reply:
Not done yet, let's do it in the csp pr